constants.go

```diff
@@ -547,17 +547,6 @@ const (
 	TraitInternalDBUsersVariable = "{{internal.db_users}}"
 )
 
-const (
-	// GSuiteIssuerURL is issuer URL used for GSuite provider
-	GSuiteIssuerURL = "https://accounts.google.com"
```

Comment:
Seems like you could keep this one?

Reply:
Just there in `constants.go`, or private in `lib/auth`?